.github/workflows/lint.yml

```diff
@@ -0,0 +1,16 @@
+name: Lint
+
+on: [push]
+
+jobs:
+  eslint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install modules
+        run: yarn
```

Comment:
Are you 100% sure that there's no need to install Node with Yarn first?  Besides, we can [cache dependencies](https://github.com/actions/setup-node#caching-packages-dependencies)  between invocations to reduce job time.

Reply:
Good point